com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -20,11 +20,18 @@ namespace UnityEngine.Rendering.Universal.Internal
     /// </summary>
     public class PostProcessPass : ScriptableRenderPass
     {
+        static readonly RTHandle k_CameraTarget = RTHandles.Alloc(BuiltinRenderTextureType.CameraTarget);
+        static readonly RTHandle k_CurrentActive = RTHandles.Alloc(BuiltinRenderTextureType.CurrentActive);
```

Comment:
No need to alloc textures for these. 
 You can get camera target from renderer and current active from BuiltinRenderTextureType.CurrentActive